packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -36,6 +36,17 @@ type BaseLinkCommonProps = {
    * @default medium
    */
   size?: 'small' | 'medium';
+  /**
+   * Defines how far your touch can start away from the link. This is a react-native only prop and has no effect on web.
+   */
+  hitSlop?:
+    | {
+        top?: number;
```

Comment:
do we have to take it as a prop for consumer? can we standardize the click-area internally without taking it from consumers?

Reply:
Ideally we should be standardising this but we haven't put any design effort on this. On the PG Mobile have they have different hitSlops for different use-cases. We'll have to take up hitSlop as a wider charter. Right now this is a blocker only for Links, hence added this but we would need to explore hitSlop support for all components in the future. Unblocking the PG team with this